### DISCUSSION TITLE
Plugins repository

### DISCUSSION THREAD

**Discussion by deku-m** [netbox-community/netbox#11557] [Ideas]
Hi,

Back in the days and now alot of plugins has been build by other people who also use netbox. And they are all listed here. 
But you notice also that alot of those plugins are only managed by one person and sometimes they fanish. And the PRs cant be added/submitted anymore.

I wonder if it isnt better to have something like the device-library and put all plugins there and everybody can work on it maintained by netbox developers/community. So everybody can contribute and still the plugins can be adjusted by the whole community. Because alot of people are doing PRs for plugins from the maintainer but doesnt add them anymore. And people need to fork it, etc.

I wish there could be something like that.

**Comment by jeremystretch** [comment]
As luck would have it, we (well, mostly @arthanson) have been working on a new website for https://netbox.dev, which will include a plugins registry where people can find and submit NetBox plugins. We hope to have the new site live in the next week or two; keep an eye out here for the announcement. We'd love to get your feedback once it launches!

**Reply by deku-m** [reply]
Good to hear, i look forward to it. Thank you for the feedback and creation of netbox.

But still my other concern about the maintainers that stop maintaining the nice plugins they have written were people sometimes make Pull Request for it. Will not be added anymore only way is then to fork ofc. I wonder if it is being maintain under netbox-community or something everybody can help. But i could be wrong.

**Reply by candlerb** [reply]
> But still my other concern about the maintainers that stop maintaining the nice plugins they have written were people sometimes make Pull Request for it. Will not be added anymore only way is then to fork ofc.

Welcome to open source software :-)

The whole point of plugins is that they are *not* maintained by the core team. The plugin interface provides a way for *other* people to extend the functionality of Netbox, without having to fork Netbox itself.
